src/js/src/index.js

```diff
@@ -4,28 +4,34 @@ import { mountLayoutWithWebSocket } from "idom-client-react";
 let LOCATION = window.location;
 let WS_PROTOCOL = "";
 if (LOCATION.protocol == "https:") {
-  WS_PROTOCOL = "wss://";
+	WS_PROTOCOL = "wss://";
```

Comment:
It looks like there's a lot of unnecessary changes here. Can you make the indent here two spaces rather than three?